test/spec.yml

```diff
@@ -457,6 +457,47 @@
         exit: 1
         err: "Empty string env 'EMPTY' in binaris.yml is not supported"
 
+- test: "concurrent execution model: deploy invoke multiple times"
+  steps:
+    -   in: |-
+            cat <<EOF >/home/dockeruser/test/binaris.yml
+            functions:
+              concurrentfunc:
+                file: function.js
+                entrypoint: handler
+                runtime: node8
+                executionModel: concurrent
+            EOF
+    -   in: |-
+            cat <<EOF >/home/dockeruser/test/function.js
+            function msleep(ms) {
+              return new Promise((resolve) => {
+                  setTimeout(resolve, ms);
+              });
+            }
+            let duringFlight = false;
```

Comment:
Maybe sleep always and hold a counter of inflight requests?
Would be nice to know how much concurrency we actually achieve

Reply:
it would be much difficult to test, now I can just do ` | grep true`